content/docs/aws/deploy/install-kubeflow.md

```diff
@@ -23,6 +23,9 @@ deploy Kubeflow on Amazon Web Services (AWS).
 
 You do not need to have an existing Amazon Elastic Container Service for Kubernetes (Amazon EKS) cluster. The deployment process will create a cluster for you.
 
+The install tools uses `eksctl` command, and doesn't support `--profile` option, now.
```

Comment:
Thanks for this update! I suggest a small change in the wording, for additional clarity:

```
The installation tool uses the `eksctl` command and doesn't support  the `--profile` option.
If you need to switch role, use the `aws sts assume-role` commands. See the AWS guide to [using temporary security credentials to request access to AWS resources](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html).
```

Reply:
Thank you for your review.
I fixed wordings.
2c40c2c